lib/config/schemas/gateway.config.json

```diff
@@ -0,0 +1,108 @@
+{
+  "$id": "http://express-gateway.io/models/gateway.config.json",
+  "type": "object",
+  "properties": {
+    "http": {
+      "type": "object",
+      "properties": {
+        "port": {
+          "type": "number"
+        }
+      }
+    },
+    "https": {
+      "type": "object",
+      "properties": {
+        "port": {
+          "type": "number"
+        },
+        "tls": {
+          "type": "object",
+          "additioanlProperties": {
+            "type": "object",
+            "properties": {
+              "key": {
+                "type": "string"
+              },
+              "cert": {
+                "type": "string"
+              }
+            }
+          }
+        }
+      }
+    },
+    "admin": {
+      "type": "object",
+      "properties": {
+        "port": {
+          "type": "number"
+        },
+        "hostname": {
+          "type": "string"
+        }
+      }
+    },
+    "apiEndpoints": {
+      "type": [
+        "object",
+        "null"
+      ],
+      "additionalProperties": {
+        "type": "object",
+        "properties": {
+          "host": {
+            "type": "string"
+          },
+          "paths": {
```

Comment:
pathRegex is also valid

Reply:
cool, I see pathRegex, 
I also remember `verbs` (or should we call it methods ) so we can skip it for now

Reply:
It's `methods` and you said it's undocumented. Do you want me to add it?

Reply:
Why is `methods` undocumented?

Reply:
I don't know guys, I wasn't part of the discussion.